src/editors/_shared/hooks/DataTypes/types/ArrayType.js

```diff
@@ -22,7 +22,7 @@ class ArrayType extends DataType {
    * @returns
    */
   validate(value) {
-    value = value?.replace(/'/g, '"');
+    value = typeof value === "string" ? value?.replace(/'/g, '"'): value;
```

Comment:
```suggestion
    value = typeof value === "string" ? value.replace(/'/g, '"'): value;
```
Don't need the `?` in `value.replace` because we already know it's of `string` type, so it won't be `null` / `undefined`.